types/favicons/index.d.ts

```diff
@@ -38,6 +38,8 @@ declare namespace favicons {
         version: string;
         /** Print logs to console? @default false */
         logging: boolean;
+        /** Use nearest neighbor resampling to preserve hard edges on pixel art @default false */
+        pixel_art: boolean;
```

Comment:
Should this be optional?  It seems like it might break existing interface implementers.

Reply:
See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1b75aeb427c14f093eba6da775df4551fe0e1efa/types/favicons/index.d.ts#L92

Reply:
Great.  Thanks!